app/dashboard/templates/ftux/skills.html

```diff
@@ -43,9 +43,9 @@ <h2>{% trans "What's Your Skill Set?" %}</h2>
     <div class="form__select2 col-12 col-md-8 col-lg-9 pl-0 pr-0">
         <select class="w-100 js-select2" id="experienceLevel">
             <option value="">{% trans "Any" %}</option>
-            <option value="beginner">{% trans "Beginner ( < 4 years Experience )" %}</option>
+            <option value="beginner">{% trans "Beginner ( < 3 years Experience )" %}</option>
```

Comment:
Isn't this inconsistent? What if I have 6 years of experience, what do I select? @PixelantDesign @thelostone-mc